api/src/test/java/org/openmrs/module/ModuleUtilIT.java

```diff
@@ -372,8 +372,8 @@ public void compareVersion_shouldCorrectlyComparingTwoVersionNumbers() throws Ex
 	 */
 	@Test
 	public void compareVersion_shouldTreatSNAPSHOTAsEarliestVersion() throws Exception {
-		String olderVersion = "1.8.3";
-		String newerVersion = "1.8.4-SNAPSHOT";
+		String olderVersion = "1.8.4~SNAPSHOT";
```

Comment:
Did you intend to remove the `-`?

Reply:
Yes, that's my bad I guess.
Should i replace it with -?

Reply:
Did you have any reason for doing so?

Reply:
I actually thought the tilde has in most cases the same definition a the hyphen in java

Reply:
And are you sure the SNAPSHOT version is evaluated older Version?

Reply:
Yes plain sure as ticket description stated it that way.

Reply:
Well I have not had time to look at the ticket description but normally SNAPSHOT is the earliest. What do you understand with this as part of this testcase name `shouldTreatSNAPSHOTAsEarliestVersion`?

Reply:
Have you gone through the ticket description to know why SNAPSHOT is made older version? :)